lib/phlex/static_component.rb

```diff
@@ -0,0 +1,24 @@
+module Phlex
+  class StaticComponent < Component
+    def call
+      return super if is_a? Compiler
+      if respond_to? :compiled_template
+        compiled_template(&@content)
+      else
+        compile_template
+        call
+      end
+    end
+
+    def compile_template
+      compiler_class = Class.new(self.class).include(Compiler)
```

Comment:
Hmm ok, but didn't exactly get why `Compiler` can not be included at compile time. I'm thinking how much it can impact runtime processing for multiple instances otherwise 🤔

Reply:
Ah yeah, I meant on boot time. Benchmark looks great!!